lib/screens/v3_api.ex

```diff
@@ -5,7 +5,7 @@ defmodule Screens.V3Api do
 
   require Logger
 
-  @default_opts [timeout: 2000, recv_timeout: 2000, hackney: [pool: :api_v3_pool]]
+  @default_opts [timeout: 2000, recv_timeout: 2000, hackney: [{:pool, :api_v3_pool}, {:checkout_timeout, 4000}]]
```

Comment:
One quick thing: `[pool: :api_v3_pool]` is syntactic sugar for `[{:pool, :api_v3_pool}]` (See [Keyword lists](https://hexdocs.pm/elixir/Keyword.html#module-examples))

I'll get back to this review soon (maybe tomorrow morning given my schedule) to double check that the option will be passed through to hackney correctly, but looks good on first glance!